test/e2e/density.go

```diff
@@ -99,20 +120,21 @@ var _ = Describe("Density", func() {
 	// Tests with "Skipped" substring in their name will be skipped when running
 	// e2e test suite without --ginkgo.focus & --ginkgo.skip flags.
 	type Density struct {
-		skip          bool
-		podsPerMinion int
+		skip           bool
+		runLatencyTest bool
+		podsPerMinion  int
 		/* Controls how often the apiserver is polled for pods */
 		interval time.Duration
 	}
 
 	densityTests := []Density{
 		// This test should always run, even if larger densities are skipped.
-		{podsPerMinion: 3, skip: false, interval: 10 * time.Second},
-		{podsPerMinion: 30, skip: false, interval: 10 * time.Second},
+		{podsPerMinion: 3, skip: false, runLatencyTest: false, interval: 10 * time.Second},
+		{podsPerMinion: 30, skip: false, runLatencyTest: false, interval: 10 * time.Second},
```

Comment:
please add a comment with how to turn this on via ginkgo (or it one has to modify the source to turn it on, please add a todo to expose it).